packages/cx-layout/doc/Header.md

```diff
@@ -52,35 +53,37 @@ const Layout: FC = ({ children }) => (
 export default Layout;
 ```
 
-#### Customing Via Shadowing (*Preferred Method)
+#### Customizing Via Shadowing (*Preferred Method)
 
 Provide the Shadowing function as defined in [Shadow](../CX_Elements/CX_Shadow).
 
 File to shadow:
 [`cxHeader.ts`](https://github.com/johnsonandjohnson/Bodiless-JS/blob/main/packages/cx-layout/src/components/Header/tokens/cxHeader.ts)
 
-#### Customing Via Extending
+#### Customizing Via Extending
 
-The site builder can create a new token that and override's with asHeaderToken() and then extend/override the specific domains.
+The Site Builder can create a new token that and overrides with `asHeaderToken()`, and then
+extends/overrides the specific domains.
```

Comment:
@hvanyo I wasn't sure what should go here:

> ...create a new token that _______ and overrides with `asHeaderToken()`, ...

I'm guessing "extends"?

Reply:
The Site Builder can create a new token that utilizes `asHeaderToken()`, and then can
extends/overrides the specific domains within that token.